cmd/node/node.go

```diff
@@ -1072,6 +1073,12 @@ func (app *SpacemeshApp) Start(*cobra.Command, []string) {
 		metrics.StartCollectingMetrics(app.Config.MetricsPort)
```

Comment:
this code starts a regular Promethues client (server for pulling the data from the node). shouldn't we NOT start it when using push metrics? we might need to extract that logic from this method that actually starts collecting..

Reply:
might need a rename to "StartMetricsServer" because apparently it doesn't actually collects the metrics but just exposes the server.

Reply:
I meant renaming the Pull component "MetricsServer" since this one is pushing and ain't a server.

Reply:
You are right.

We have a server to pull metrics from, function that start it is called `StartCollectingMetrics`. It could be renamed to `StartMetricsServer`.

Pushing metrics starts with `StartMetricsServer`. Let's rename it to `StartPushingMetrics`.